aws/lambda-ms-teams-notifier/README.md

```diff
@@ -0,0 +1,55 @@
+# Terraform Module: MS Teams Notifier
+
+## Table of Contents
+
+1. [Scope](#scope)
+2. [Used Services & Features](#services-and-features)
+3. [Usage](#usage)
+4. [Input](#input)
+5. [Output](#output)
+6. [Additional](#additional)
+7. [Roadmap](#roadmap)
+    
+## Scope
+This function is just a notifier which pushes Cloudwatch Alarms messages (triggered via SNS) to MS Teams by using the channel's hook url.
+    
+## Used Services & Features
+* AWS Lambda including IAM Role and Policy
+* Python 3.8 (no external libs used)
+
+## Usage
+```
+module "ms-team-notifier-lambda" {
+    source = "git@github.com:codecampn/terraform-modules-aws-lambda-ms-teams-notifier.git"
```

Comment:
Wrong implementation link is shown. How will the source be included in the future?

Reply:
solved with current commit